kotlin/services/lambda/Readme.md

```diff
@@ -1,22 +1,108 @@
-# AWS Lambda Kotlin code examples
+# AWS Lambda code examples for the SDK for Kotlin
```

Comment:
AWS Lambda --> Lambda 
(OK to use short service name in titles)

Reply:
fixed